ecs-init/config/common.go

```diff
@@ -133,3 +137,15 @@ func getBaseLocationForRegion(regionName string) string {
 
 	return s3BucketURL
 }
+
+// GetCgroupMountpoint returns the cgroup mountpoint
+func GetCgroupMountpoint() (string, error) {
```

Comment:
Is this resolution of the mountpoint outside or inside the container here? Should we call this something like `GetHostCgroupPath`?

Reply:
The resolution is outside the container, so I think the current name makes sense.